packages/core/src/checkout-buttons/strategies/amazon-pay-v2/amazon-pay-v2-button-options.ts

```diff
@@ -1,6 +1,6 @@
-import { AmazonPayV2ButtonParams } from '../../../payment/strategies/amazon-pay-v2';
+import { AmazonPayV2ButtonParams, AmazonPayV2Ph4ButtonParams } from '../../../payment/strategies/amazon-pay-v2';
```

Comment:
Can we find a better name for this as compared to `AmazonPayV2Ph4ButtonParams`? Something to explain what `Ph` is

Reply:
What about `AmazonPayV2Phase4ButtonParams`, `AmazonPayV2ApiV2ButtonParams` or `AmazonPayV2NewButtonParams`? 🤪